markdown/ch4.markdown

```diff
@@ -291,7 +290,7 @@ freqs, times, Sx = signal.spectrogram(audio, fs=rate, window='hanning',
                                       detrend=False, scaling='spectrum')
 
 ## Plot using:
-# plt.pcolormesh(times, freqs, 10 * np.log10(Sx), cmap='viridis');
+# plt.pcolormesh(times, freqs, 10 * np.log10(Sx));
```

Comment:
I'd say it's worth plotting. It illustrates that it is the same thing. (This was not clear to me, even reading the context. I think it doesn't help that you used imshow above and pcolormesh here.)